include/subgroup/tile/impl/op_function.hpp

```diff
@@ -676,4 +676,22 @@ layout_convert(T_dst& dst, T_src& src) {
     }
   }
 }
+
+template <typename T>
+void dump_mat(
+    T mat,
+    size_t tile_x = T::tile_size_x,
+    size_t tile_y = T::tile_size_y) {
+#pragma unroll
+  for (size_t row = 0; row < tile_x; row++) {
```

Comment:
emmm, swap the tile_x & tile_y

Reply:
fix
#pragma unroll
  for (size_t row = 0; row < tile_y; row++) {
#pragma unroll
    for (size_t col = 0; col < tile_x; col++) {